build.gradle

```diff
@@ -1 +1,34 @@
-ant.importBuild 'build.xml'
\ No newline at end of file
+apply plugin: 'java'
+apply plugin:'application'
+mainClassName = "edu.ucsb.cs56.projects.utilities.credit_card_validator"
```

Comment:
This line should be:
`mainClassName = "edu.ucsb.cs56.projects.utilities.credit_card_validator.Menu"`